ix-dev/community/immich/app.yaml

```diff
@@ -45,4 +45,4 @@ sources:
 - https://github.com/immich-app/immich
 title: Immich
 train: community
-version: 1.5.1
```

Comment:
Hmm it seems that is bumped multiple times, where it shouldnt. Needs to be checked